examples/zerolog/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/airbrake/gobrake/v5"
+	airbrakeWriter "github.com/airbrake/gobrake/v5/zerolog"
+	"github.com/rs/zerolog"
+)
+
+var ProjectId int64 = 999999                               // Insert your Project Id here
+var ProjectKey string = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" // Insert your Project Key here
+
+func main() {
+	airbrake := gobrake.NewNotifierWithOptions(&gobrake.NotifierOptions{
+		ProjectId:   ProjectId,
+		ProjectKey:  ProjectKey,
+		Environment: "production",
+	})
+
+	defer airbrake.Close()
+
+	// Note: This writer only accepts errors, fatal and panic logs, all others will be ignored.
+	// You can still send logs to stdout or another writer via io.MultiWriter
+	w := airbrakeWriter.New(airbrake)
+
+	// Insert the newly created airbrakeWriter into zerolog
+	log := zerolog.New(io.MultiWriter(os.Stdout, w))
+	zerolog.SetGlobalLevel(zerolog.DebugLevel)
+
+	// Creates a sub logger that has additional ctx
+	loggerWithData := log.With().
+		Dict("ctx",
+			zerolog.Dict().
+				Str("BUILD", "gitsha").
+				Str("VERSION", "").
+				Str("TIME", ""),
+		).
+		Str("file", "something.png").
+		Str("type", "image/png").
+		Str("user", "tobi").
+		Logger()
+
+	fmt.Printf("Check your Airbrake dashboard at https://airbrake.io/projects/%v to see these log messages\n", ProjectId)
```

Comment:
I just copied this line from the apexlogs example. I can update both if you want.

Reply:
Yes, please update both locations.

Reply:
@scottsbaldwin what would be better? Can you give me an example of what you want?

Reply:
```suggestion
	fmt.Printf("Check your Airbrake dashboard at https://YOUR_SUBDOMAIN.airbrake.io/projects/%v to see these error occurrences\n", ProjectId)
```

This will address Scott's comments if you want to use it. It won't quite work as a clickable link but it's how we want users to access their dashboard. 

Bonus would be to get the locate link (what gets spit out from our API after you send the error).